internal/http/services/sciencemesh/sciencemesh.go

```diff
@@ -108,7 +113,7 @@ func (s *svc) Prefix() string {
 }
 
 func (s *svc) Unprotected() []string {
-	return nil
+	return []string{"/create-share"}
```

Comment:
Why Unprotected? All endpoints including this one should be protected by HTTP Basic Auth with username/password which gets checked by the Auth Manager, right?

Reply:
Because the original "/ocm/send" has some auth parameters in the request. To keep it compatible i left the authentication in the /create-share request. If it's protected this requires double auth.
But I'm fine we can remove the authentication phase in this endpoint, and leave it protected.

Reply:
Reviving this: I fully support removing the "custom auth" implemented via auth parameters in the request payload. It was there for some legacy reasons but it makes more sense to use the auth interceptors and make the endpoint protected.